backend/src/endpoints/users/users.routes.ts

```diff
@@ -19,4 +19,7 @@ router.get("/validation/:value", UsersController.getUsersByValidationValue);
 // Create a user
 router.post("/", UsersController.createUser);
 
+// Updates a user's score.
+router.post("/username/:username/updateprogress", UsersController.updateUserProgress);
```

Comment:
Never, ever set an endpoint to an action like this. It should be a noun -- /username/:username/progress, with GET returning the values and POST/PUT/PATCH/whatever writing the values, and potentially DELETE clearing them to baseline.

Reply:
changed function name to postUserProgressByUsername.

changed function in UsersData to postContentProgressByUsername, postLessonProgressByUsername (resp.)

Reply:
Hey sorry I forgot to push changes.